gateway-ha/gateway-ha-config.yml

```diff
@@ -6,7 +6,7 @@ requestRouter:
   port: 8080
   name: trinoRouter
   historySize: 1000
-  requestBufferSize: 8 * 1024
+  requestBufferSize: 8195
```

Comment:
question: how come we opted into 8195 instead of 8192?

Reply:
It's a typo, thanks for catching it.